scripts/package-build/vpp/package.toml

```diff
@@ -5,6 +5,13 @@ scm_url = "https://github.com/vyos/vyos-vpp-patches"
 build_cmd = "/bin/true"
 apply_patches = false
 
+[[packages]]
+name = "vpp-sflow"
+commit_id = "v0.9.02-2"
```

Comment:
Rebase please an dump to version `v1.0.01-1`

Reply:
Rebased and version bumped.